xml/System.Collections.Generic/HashSet`1.xml

```diff
@@ -1309,10 +1309,10 @@ The following example demonstrates how to merge two disparate sets. This example
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="TAlternate">To be added.</typeparam>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TAlternate">The alternate type of instance for performing lookups.</typeparam>
+        <summary>Gets an instance of a type that may be used to perform operations on the current <see cref="T:System.Collections.Generic.HashSet`1"/> using a <typeparamref name="TAlternate"/> instead of a <typeparamref name="T"/>.</summary>
```

Comment:
```suggestion
        <summary>Gets an instance of a type that can be used to perform operations on the current <see cref="T:System.Collections.Generic.HashSet`1"/> using a <typeparamref name="TAlternate"/> instead of a <typeparamref name="T"/>.</summary>
```